packages/zowe-explorer/src/globals.ts

```diff
@@ -93,15 +93,17 @@ export const EXTENDER_CONFIG: imperative.ICommandProfileTypeConfiguration[] = []
 export const ZOWE_CLI_SCM = "@zowe/cli";
 export const MAX_DATASET_LENGTH = 44;
 export const MAX_MEMBER_LENGTH = 8;
+// eslint-disable-next-line no-useless-escape
 export const DS_NAME_REGEX_CHECK = /^[a-zA-Z#@\$][a-zA-Z0-9#@\$\-]{0,7}(\.[a-zA-Z#@\$][a-zA-Z0-9#@\$\-]{0,7})*$/;
+// eslint-disable-next-line no-useless-escape
 export const MEMBER_NAME_REGEX_CHECK = /^[a-zA-Z#@\$][a-zA-Z0-9#@\$]{0,7}$/;
 export let ACTIVATED = false;
 export let PROFILE_SECURITY: string | boolean = ZOWE_CLI_SCM;
 export let SAVED_PROFILE_CONTENTS = new Uint8Array();
 export const JOBS_MAX_PREFIX = 8;
 
 // Dictionary describing translation from old configuration names to new standardized names
-export const configurationDictionary = {
+export const configurationDictionary: { [k: string]: string } = {
```

Comment:
Curious if we have a preference for `{ [k: string]: string }` vs `Record<string, string>`?